Sources/hostmgr/commands/vm/VMStop.swift

```diff
@@ -19,12 +19,17 @@ struct VMStopCommand: ParsableCommand {
     var all: Bool = false
 
     func run() throws {
-        try all ? shutdownAll() : virtualMachine?.shutdown()
-    }
 
-    private func shutdownAll() throws {
-        try Parallels()
-            .lookupRunningVMs()
-            .forEach { try $0.shutdown(immediately: immediately) }
+        if all {
+            try libhostmgr.stopAllRunningVMs(immediately: self.immediately)
+            Console.exit()
```

Comment:
Would it make sense to add a `return` here, just in case something changes in `Console.exit()`?

If so, what do you think of using `guard all == false else ...` to ensure `return` is called?

Reply:
Hmm interesting – I wouldn't expect `Console.exit()` to ever do anything except stop the application. [It has the return type `Never`](https://github.com/Automattic/hostmgr/blob/996301c468aa4d0f33ba37ebbe8a39dd302ba803/Sources/libhostmgr/CLI/Console.swift#L186-L189), so it's just as valid as `throws` for breaking out of the current scope, and the type system ensures that we can't accidentally change this.

That said, using `guard` here would allow the type system to enforce this for us, so I've done that in b23a49eb7f9ed014e18d4c67a6b22b233b393953.